src/Spectre.libGaClassifier/Spectre.libGaClassifier.vcxproj

```diff
@@ -21,8 +21,8 @@
   </ItemGroup>
   <PropertyGroup Label="Globals">
     <VCProjectVersion>15.0</VCProjectVersion>
-    <ProjectGuid>{ACC9B0E5-4106-4392-A7B6-A8A050759AFB}</ProjectGuid>
-    <RootNamespace>SpectreGaSvmNative</RootNamespace>
+    <ProjectGuid>{9FAF96E9-1983-4BD3-8C5C-FB922AC73126}</ProjectGuid>
+    <RootNamespace>SpectrelibGaClassifier</RootNamespace>
```

Comment:
There should be `Spectre::libGaClassifier`.

Reply:
done

Reply:
Hmmm... I've made a mistake. It should actually be something like `spectre::supervised::gaclassifier`...